### PR TITLE
Add project type icons

### PR DESCRIPTION
**Pull request by prachigarg19**

### CODE REVIEW

File: src/features/projects/components/ProjectTypeIcon.tsx [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import NaturalRegenerationIcon from '../../../../public/assets/images/icons/project/NaturalRegenerationIcon';
+interface Props {
+  projectType: string;
+}
+const ProjectTypeIcon = ({ projectType }: Props) => {
+  switch (projectType) {
+    case 'natural-regeneration':
+      return <NaturalRegenerationIcon />;
```
Comment (norbertschuler) [inline src/features/projects/components/ProjectTypeIcon.tsx:9]: Do we have any example project on staging to see how this looks like?

**Comment by norbertschuler** [comment]
Be aware this PR might get into conflict with https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1794 :-)

File: src/features/projects/components/ProjectTypeIcon.tsx (hunk 1 above)
Comment (norbertschuler) [reply src/features/projects/components/ProjectTypeIcon.tsx]: Now I see some :-)
Bildschirmfoto 2023-06-10 um 12 09 30

**Review by norbertschuler — APPROVED** [review]
LGFM

File: src/features/projects/components/maps/Markers.tsx [hunk 2]
```diff
@@ -32,7 +33,13 @@ export default function Markers({
   const handleClose = () => {
     setOpen(false);
   };
-
+  const markerBackgroundColor = (project) => {
+    return project.isTopProject
```
Comment (norbertschuler) [inline src/features/projects/components/maps/Markers.tsx:37]: How about using `project.isTopProject && project.isApproved` as at https://github.com/Plant-for-the-Planet-org/planet-webapp/pull/1794 or is `project.isApproved` for this coloring use case?

**Review by norbertschuler — APPROVED** [review]

**Comment by mariahosfeld** [comment]
The icons for "restoration" are missing, use the same ones as for "Large Scale Restoration"

**Comment by norbertschuler** [comment]
I merged develop into this PR for better testing. Still seems to work ;-)

**Comment by mariahosfeld** [comment]
@prachigarg19 , please use blue markers for conservation projects.

**Pull request merged**